### DISCUSSION TITLE
📚 Add your testimonials for the community

### DISCUSSION THREAD

**Discussion by Panquesito7** [WebXDAO/WebXDAO.github.io#50] [General]
Welcome to the WebXDAO! 👋 

For our website, we're going to add some testimonials
of what do you think about our community and resources.

Write below and let us know what do you think! 👇

**Comment by suitupstan** [comment]
Web3Community is a awesome community of like-minded features! Great guidance and support here! Thanks for every single opportunity, you've given to me!

**Comment by suitupstan** [comment]
This is a community I don't have enough words to appreciate, WebXDao has really supportive members who helped me build my first DAPP, they also gave me a chance to participate with their founders in hackathons twice! They gave me opportunities, guidance, resources, experience and a lot more! 1000/10 recommended community for Web3! :100:

**Reply by suitupstan** [reply]
New review!!^^ v-WebXDAO
